spec/lib/worker_suite_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_helper'
+
+describe LogStash::PerformanceMeter::Core do
+
+  let(:config)        { 'spec/fixtures/config.yml' }
+  let(:logstash_home) { '.' }
+  let(:suite_def)     { 'spec/fixtures/worker_basic_suite.rb' }
+  let(:serial_runner) { double('DummySerialRunner') }
+  let(:runner)        { LogStash::PerformanceMeter::Runner }
+
+  let(:run_outcome)   { { :percentile => [2000] , :elapsed => 100, :events_count => 3000, :start_time => 12 } }
+  subject(:manager) { LogStash::PerformanceMeter::Core.new(suite_def, logstash_home, config, runner) }
+
+  context "with a valid configuration and worker threads set to 2" do
```

Comment:
can we make the number of workers a variable here? by using a fixture, number of workers used is more hidden, just want to make sure people understand.